src/main/resources/static/common/js/approver.js

```diff
@@ -131,7 +131,25 @@ app.controller('approverHomePageController', function($rootScope, $http, $filter
             approver.selectPage(approver.currentPage);
             approver.showOrderDetail['show'] = false;
         }).catch(function(result) {
-            console.log(result);
+            var errCode = result.status;
+            var errMsg;
+            switch (errCode) {
+                case 400:
+                    errMsg = $filter('translate')('NON_REPEATABLE_OPERATIONS');
+                    break;
+                case 401:
+                    errMsg = $filter('translate')('NO_AUTHORIZATION_TO_UPDATE_ORDER');
+                    break;
+                case 403:
+                    errMsg = $filter('translate')('NO_PERMISSION_TO_UPDATE_ORDER');
+                    break;
+                case 404:
+                    errMsg = $filter('translate')('ORDER_NOT_FOUND');
+                    break;
+                default:
+                    errMsg = $filter('translate')('COMMENTS_TOO_LONG');
```

Comment:
I think the default error message should not related to comments too long. We fixed this in android side in another bug. Take a look at the fix for scenario 2 in https://jira.parasoft.com/browse/PDA-1152.